src/frontend/components/JoinDiscussionAskButton/index.tsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
```

Comment:
Did you forget to localize the form?

Reply:
Yes I forget to translate everything !!! Sorry